lib/bolt/config/options.rb

```diff
@@ -34,6 +34,20 @@ module Options
               description: "The name of the plugin.",
               type: "string"
             }
+          },
+          additionalProperties: {
+            "_cache" => {
```

Comment:
This should be under `:properties` since it's a known property. `:additionalProperties` is meant to validate properties that are unknown (such as in `log` where we don't know the filenames used for keys, but we do know that they should be hashes with only two possible keys).

Reply:
Hm ok...this was raising a "_cache required" error, but I think I solved that in the code itself.